docs/src/pages/configurations/overview/index.md

```diff
@@ -0,0 +1,62 @@
+
+---
+id: 'overview'
+title: 'Configuration overview'
+---
+
+For CLI options see: [here](/docs/cli-options).
+
+> migration guide: This page document the method to configure storybook introduced recently in 5.3.0, consult the [migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md) if you want to migrate to this format of configuring storybook.
+
+## main configuration
+
+Storybook has a few files it uses for configuration, and they are grouped together into a directory (default: `.storybook`).
+
+The most import file is the `main.js` file. This is where general config is declared.
+
+Here's an minimal example of a that file:
+
+```js
+module.exports = {
+  stories: ['../src/components/**/*.stories.js'],
+  addons: [
+    '@storybook/addon-essentials',
+  ],
+};
+```
+
+The addons field can refer to traditional [addons](docs/addons/introduction), but it can also include [presets](/docs/presets/introduction/) extending the config further.
+
+## manager & preview
```

Comment:
Can we have at least a minimal example of what each file looks like?

Reply:
✔️